pyam/core.py

```diff
@@ -957,6 +959,28 @@ def validate(self, criteria={}, exclude_on_fail=False):
                 self._exclude_on_fail(df)
             return df.reset_index()
 
+    def compute_bias(self, name, method, axis):
+        """Compute the bias weights and add to 'meta'
+
+        Parameters
+        ----------
+        name : str
+           Column name in the 'meta' dataframe
+        method : str
+            Method to compute the bias weights, see the notes
+        axis : str
+            Index dimensions on which to apply the `method`
```

Comment:
I think this documentation is not the easiest to follow - as it's currently defined (without wildcard matching) only "model" or "scenario" are sensible choices of axis, so maybe giving an example would be useful.

Reply:
Added the example from the description of this PR to the notes section of this method